pkg/controllers/managementuser/resourcequota/resource_quota_sync.go

```diff
@@ -257,12 +243,12 @@ func (c *SyncController) getExistingLimitRange(ns *corev1.Namespace) (*corev1.Li
 	return limitRanger[0], nil
 }
 
-func (c *SyncController) getNamespaceResourceQuota(ns *corev1.Namespace) (v32.NamespaceResourceQuota, error) {
+func (c *SyncController) getNamespaceResourceQuota(ns *corev1.Namespace) (v32.ResourceQuotaLimit, error) {
```

Comment:
Now that this returns a Limit you may want to change the name of this function to indicate that this is returning a limit (especially given how much we swap between the two in this file).